source/lambda/helper/python/kendraHelper.py

```diff
@@ -0,0 +1,162 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+from helper import S3Helper
+
+
+class KendraHelper:
+    
+
+    #
+    #   This function instruct Kendra to index a pdf document in s3
+    #
+    #   region:                 region of Kendra index
+    #   kendraIndexId:          Kendra index id
+    #   kendraRoleArn:          a role that Kendra can assume to read the s3 bucket
+    #   s3bucket:               bucket name where document to index exists
+    #   s3key:                  key of the document to index
+    #   documentId:             the document id generated by DUS
+    #
+    def indexDocument(self,
+                      region,
+                      kendraIndexId,
+                      kendraRoleArn,
+                      s3bucket,
```

Comment:
Please add AccessControlList as a non-required input parameter to indexDocument. When no parameter is passed, we can default to the everynody group. However, when we look to make changes in the future to allow users to provide tags for uploading docs, we need to provide the capacbility